README.md

```diff
@@ -1,71 +1,43 @@
-Terraform Provider
-==================
+# To update Terrform-provider-chef
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+## Requirement
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+Install gpg, go, sha256sum, goreleaser if you don't have them yet:
```

Comment:
We could also check that some these exist at the top of the publish script (I think just gpg and sha256sum as we will already have built the artifacts by then)